test/evergreen_develop.yml

```diff
@@ -61,6 +61,7 @@ buildvariants:
     - ubuntu2004-medium
   expansions:
     <<: *ubuntu2004-perf-tests-expansions-template
+    database: WTPerformanceDataDB
```

Comment:
I think it is better to add `database: WTPerformanceDataDB` from here to ubuntu2004-perf-tests-expansions-template to avoid redundancy.

Reply:
Fixed in fae386d70a3885cebadc7bb0f0693f73b5f4f37e